Demo/Demo/PayPalWebPayments/PayPalWebViewModel/PayPalWebViewModel.swift

```diff
@@ -100,47 +155,80 @@ class PayPalWebViewModel: ObservableObject {
             payPalDataCollector = PayPalDataCollector(config: config)
             return payPalClient
         } catch {
-            updateState(.error(message: error.localizedDescription))
+            DispatchQueue.main.async {
+                self.state.createdOrderResponse = .error(message: error.localizedDescription)
+            }
             print("❌ failed to create PayPalWebCheckoutClient with error: \(error.localizedDescription)")
             return nil
         }
     }
 
     func completeTransaction() async throws {
         do {
-            updateState(.loading)
+            setLoadingState()
+            
+            try await processIntent(orderID: orderID ?? "")
```

Comment:
I think you are calling the auth/capture twice.
once here and in completeOrder

Reply:
Gotcha, in this case, authorizeOrder and captureOrder are no longer needed since completeOrder is verifying the intent. these two methods can be removed.

Reply:
I mean your processIntent function and call should be removed